packages/salesforcedx-apex-replay-debugger/src/breakpoints/breakpointUtil.ts

```diff
@@ -9,32 +9,11 @@ import { LineBreakpointInfo } from '@salesforce/salesforcedx-utils';
 import { DebugProtocol } from '@vscode/debugprotocol';
 
 export class BreakpointUtil {
-  private static instance: BreakpointUtil;
```

Comment:
a bunch of this was either completely unused or used only for testing.


exposing the 2 maps simplifies things a lot